components/movementsensor/gpsrtkpmtk/gpsrtkpmtk_test.go

```diff
@@ -0,0 +1,189 @@
+package gpsrtkpmtk
+
+import (
+	"context"
+	"math"
+	"testing"
+
+	"github.com/edaniels/golog"
+	geo "github.com/kellydunn/golang-geo"
+	"go.viam.com/test"
+	"go.viam.com/utils"
+
+	"go.viam.com/rdk/components/movementsensor/fake"
+	rtk "go.viam.com/rdk/components/movementsensor/rtkutils"
+)
+
+var (
```

Comment:
I think it's clearer to define these variables in `TestPosition` instead since that's where they are being used